Documentation/ceph-object-store-crd.md

```diff
@@ -162,3 +162,18 @@ The endpoint health check procedure is the following:
 6. Update CR health status check
 
 Rook-Ceph always keeps the bucket and the user for the health check, it just does a PUT and GET of an s3 object since creating a bucket is an expensive operation.
+
+## Security settings
```

Comment:
We should mention the `security` option here like we did for `healthCheck` above. Also, we should make it more clear where the `VAULT_SECRET_ENGINE` should belong under the `security` tree.

```
* `security`
  * `kms`
    * `VAULT_SECRET_ENGINE` option ....
```